test/suite/hook_tiered.py

```diff
@@ -87,12 +135,25 @@ def wiredtiger_open_tiered(ignored_self, args):
 
     return args
 
+# We want readonly tests to run with tiered storage, since it is possible to do readonly
+# operations.  This function is called for two purposes:
+#  - when readonly is enabled, we don't want to do flush_tier calls.
+#  - normally the hook silently removes other (not supported) calls, like alter/compact/drop.
```

Comment:
Monica and Alexey added some support for alter and drop in the tiered table schema project.